docs/standard/security/fips-compliance.md

```diff
@@ -0,0 +1,30 @@
+---
+title: .NET Core FIPS compliance
+description: Explains .NET Core FIPS compliance
+ms.date: 11/20/2019
+author: "Rick-Anderson"
+ms.author: "riande"
+---
+
+# .NET Core FIPS compliance
+
+.NET Core:
+
+* Does **not** implement any cryptographic primitives.
+* Passes cryptographic primitives calls through to the standard modules the underlying operating system provides.
+* Does **not** enforce FIPS compliance in .NET Core apps.
+* Does **not** stop developers from using non-FIPS compliant algorithms, even on systems configured for FIPS compliance.
+
+It's the responsibility of the system administrator to configure FIPS compliance for an operating system.
+
+If code is written for a FIPS-compliant environment, it's the responsibility of the developer to esure that:
+
+* Non-compliant FIPS algorithms are not used.
+* FIPS compliance is met.
```

Comment:
Not sure this line is needed, especially as the developer can't ensure that, it's a property of the OS configuration

Reply:
I added that to avoid the dreaded one-line bullet list. My D&I commitment is accessibility, and long sentences don't MT (machine translate). What's left is OK.

Reply:
Ah that explains why my lovely formal writing has gone :)